src/Equations/acoustic/Kernels/DeviceAux/KernelsAux.h

```diff
@@ -0,0 +1,108 @@
+#pragma once
+
+#include "Kernels/precision.hpp"
+#include "generated_code/init.h"
+
```

Comment:
Are these kernels symlinks? It's best to just create a symlink to the whole elastic kernel folder—as they should be exactly the same.